File: Larger Product or Sum.js
function sumOrProduct(array, n) {
  const arr_s = [...array].sort((a,b) => a-b)
  const ln = arr_s.slice(-n,).reduce((a,b) => a+b)
  const sn = arr_s.slice(0,n).reduce((a,b) => a*b)
  if(sn>ln){
    return "product"
  }else if(sn<ln){
    return "sum"
  }else {
    return "same"
  }
}

File: Larger Product or Sum.py
from functools import reduce
def sum_or_product(array, n):
    arr_s = sorted(array)
    ln = reduce(lambda a,b:a+b,arr_s[-n:])
    sn = reduce(lambda a,b:a*b,arr_s[:n])
    if ln>sn:
        return "sum"
    elif ln<sn:
        return "product"
    else:
        return "same"
    
    
